projects/rv/connect-plugin/hook/ftrack_rv_launcher.py

```diff
@@ -144,7 +144,7 @@ def _discover_applications(self):
             applications.extend(
                 self._search_filesystem(
                     expression=prefix
-                    + ['[Tweak|Shotgun|ShotGrid]', 'RV.\d.+', 'bin', 'rv.exe'],
+                    + ['[Autodesk|Tweak|Shotgun|ShotGrid]', 'RV.\d.+', 'bin', 'rv.exe'],
```

Comment:
Could this by now also be `Flow`?

Reply:
I haven't fond it yet (maybe in a future ? ), and I've been downlading the latest available.
![image](https://github.com/user-attachments/assets/934c874d-ae65-43f7-a266-04a01b97eb21)